### ISSUE TITLE
Warn about duplicate links in the "links list"

### ISSUE THREAD

**Issue by matatk** [ThePacielloGroup/WebAccessibilityToolbar#101]
This could look at the accessible link text (i.e. text content of the element(s) / alt text / other?) and warn the user if [WCAG 2.4.4 (Link Purpose in Context)](http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html) has been violated.

**Comment by patrickhlauke** [comment]
Going to close this as part of an upcoming feature freeze
